app/controllers/site_channels_controller.rb

```diff
@@ -135,7 +135,7 @@ def require_unverified_site
 
   def require_https_enabled_site
     return if current_channel.details.supports_https?
-    redirect_to(site_last_verification_method_path(current_channel.details), alert: t(".alert"))
+    redirect_to(site_last_verification_method_path(channel=current_channel), alert: t(".alert"))
```

Comment:
This change will yield a missing translation.

![image](https://user-images.githubusercontent.com/12549658/39763701-dcd4d5ae-52ab-11e8-86a9-7974e219b852.png)